Changes.md

```diff
@@ -1,6 +1,11 @@
 1.3.x.x (relative to 1.3.6.0)
 =======
 
+Features
+--------
+
+- Viewer : Added the ability to send a snapshot of the current viewport to a catalogue.
```

Comment:
Might be worth mentioning that this is available from the right-click menu?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5533/commits/d2245108847a725a76943d32f84d38b3cff59418